env/project.yml

```diff
@@ -37,3 +38,4 @@ engines:
   tk-nukestudio: "@settings.tk-nukestudio.shot"
   tk-shell: "@settings.tk-shell"
   tk-shotgun: "@settings.tk-shotgun.task"
+  tk-silhouette: "@settings.tk-silhouette.task"
```

Comment:
We're using silhouette at the project level?? Unless its being used for Editorial/Conform, that seems wrong to me. Same for 3de actually.

Reply:
Well, that would then apply to every engine except tk-nukestudio and tk-shell....Do we really want to restrict it like that?
If context change fails (for some reason), things usually fall back to project context. I'm not sure what the consequence of the engine not being available then would be.

Reply:
AFAIK, most DCC's startup in (and fall back to) the `base` environment, not the `project` environment. I suppose there's no harm in keeping this here. What's more important is limiting access to things like the publisher when outside a task context, so feel free to ignore my comment on this. All good!